data/hai missions.txt

```diff
@@ -435,3 +435,566 @@ mission "Unfettered returning home"
+		fleet "Small Northern Pirate"
+	npc
+		personality staying
+		government "Pirate"
+		system "Rajak"
+		fleet "Large Northern Pirate"
+	npc
+		personality staying
+		government "Pirate"
+		system destination
+		fleet "Large Northern Pirate"
+
+mission "Expanding Business [3]"
+	landing
+	name "An Interesting Proposition"
+	description "Escort Turner's ship to <destination> so that he can speak with an old friend."
+	destination "Rust"
+	to offer
+		has "Expanding Business [2]: done"
+	on offer
+		conversation
+			`Turner jumps off of his Star Queen before the boarding ramp fully reaches the ground, clearly eager to get moving with this project.`
+			`	Turner swears under his breath when he walks up to you. "Low life pirates giving us trouble on the way here. That's the kind of trouble I'm talking about that an outfitter in Hai space would help with. There's no telling what would have happened had my Bulk Freighters not had their missiles. Or if you were not there to protect them, of course."`
+			`	Turner has a talk with the spaceport authorities, and within a short amount of time the necessary cargo is loaded on to the freighters. When the freighters are fully loaded, they take off without even giving you time to get to your ship.`
+			`	"They can get back to Greenwater on their own," Turner says from behind you as you watch the freighters fly off. "I told them to leave as soon as they could. As for you and me, I need escort to <destination>. I received a message from an old friend there who would like to have a chat with me ASAP."`
+				accept
+				
+	npc save
+		personality escort
+		government "Merchant"
+		ship "Star Queen (Hai)" "Chryso Vasilia"
+
+mission "Expanding Business [4]"
+	landing
+	name "An Interesting Proposition"
+	description "Escort Turner back to <destination> to oversee the construction of the outfitter."
+	destination "Greenwater"
+	to offer
+		has "Expanding Business [3]: done"
+	on offer
+		conversation
+			`Turner lands his ship outside of the Kraz Cybernetics building. "Wait here while I have a talk with my friend, Edward. This should only take a few minutes."`
+			`	A few hours later, Turner returns carrying a black suitcase. "Sorry, that took longer than I expected. Now, let's get back to <destination>."`
+				accept
+			
+	on complete
+		event "outfitter on greenwater" 95
+		conversation
+			`By the time you finally reach <planet>, construction on the outfitter is already underway. The construction workers have laid the foundation of the outfitter close to the spaceport, taking up a good portion of a park.`
+			`	"I had to pay a pretty penny to get the outfitter this close to the spaceport," Turner says to you, "but it'll all be worth it in the end."`
+			`	"The outfitter should be done in three months or so. You'll know it's done once you find an extra five thousand credits in your pocket every day." Turner pats you on the back. "Have fun until then, Captain <last>. Keep yourself busy saving the galaxy."`
```

Comment:
`once` -> `when` ?

Reply:
Given that this is a character speaking, I'm less inclined to change it to something that could be "more correct."